gui/src/main/java/org/jboss/as/console/mbui/widgets/ModelNodeFormBuilder.java

```diff
@@ -649,17 +649,23 @@ public int compare(Property property, Property property1) {
                     // iterate over the "requires" attribute list
 
                     boolean sourceItemDefined = isFormItemDefined(sourceFormItem);
```

Comment:
Hi Michel, the problem is in the isFormItemDefined(sourceFormItem) method. 

The "item" parameter may be null when using MBUI, so check it for null and will be safe, as below:
     private boolean isFormItemDefined(FormItem item) {
        boolean defined = false;
        if (item != null) {
           // code
        }
        return defined;